crates/tests/Cargo.toml

```diff
@@ -1,13 +1,11 @@
 [package]
 name = "vello_tests"
+description = "Helper code for writing Vello tests."
 edition.workspace = true
-version.workspace = true
```

Comment:
I didn't know we could have a package without a version - that's neat.

Reply:
Yeah it's a new feature since 1.75. Non-published crates don't benefit from a version so now Cargo supports reducing that noise.

Reply:
We should bring this to all the applicable crates. I think `with_winit` still has a version, for example.

Reply:
Oh, I see your commit

Reply:
I agree and `with_winit` originally had this change too. However turns out that `cargo-apk` still has the old requirement, perhaps because it depends on some older fork of Cargo? I didn't investigate deeply and for now just [added the version back to `with_winit`](https://github.com/linebender/vello/pull/478/commits/88b8ac19afc5dd1f0c861411e7b41ba94fb5847c).

Hopefully `cargo-apk` will get the update, although their warning
> This tool is deprecated in favour of [xbuild](https://github.com/rust-mobile/xbuild).

doesn't exactly give me high confidence.

Maybe we can get them updated, maybe we move to xbuild (?), maybe we need to add version back to even some other examples just for the sake of Android.

Reply:
I looked a bit deeper into this. The issue comes directly from `cargo-apk`'s custom `Cargo.toml` parser. I submitted a patch to them ([cargo-apk#56](https://github.com/rust-mobile/cargo-apk/pull/56)), let's see if and when they accept it.